App.js

```diff
@@ -1,20 +1,28 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { StatusBar } from "expo-status-bar";
+import { StyleSheet, Text, View } from "react-native";
+import { NavigationContainer } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import Menu from "./src/components/Menu/Menu";
+import PassPlay from "./src/components/PassPlay/PassPlay";
+
+const Stack = createNativeStackNavigator();
 
 export default function App() {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen name="Menu" component={Menu} options={{ title: "Menu" }} />
+        <Stack.Screen name="PassPlay" component={PassPlay} options={{ title: "PassPlay", headerTintColor: "black" }} />
```

Comment:
It is a bad approach to pass objects to nested components, especially to screen components as it may cause multiple unneeded re-renders what can lead to multiple issues